i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-kvm-avocent-acs-8000-snmp.md

```diff
@@ -0,0 +1,509 @@
+| --filter             | Exclude the items given as a comma-separated list (example: --filter=psu). You can also exclude items from specific instances: --filter=psu,1                                                                                       |
+| --absent-problem     | Return an error if an entity is not 'present' (default is skipping) (comma separated list). Can be specific or global: --absent-problem=psu                                                                                          |
+| --no-component       | Define the expected status if no components are found (default: critical).                                                                                                                                                          |
+| --threshold-overload | Use this option to override the status returned by the plugin when the status label matches a regular expression (syntax: section,\[instance,\]status,regexp). Example: --threshold-overload='psu,CRITICAL,^(?!(statePowerOn)$)'    |
+
+</TabItem>
+<TabItem value="Load" label="Load">
+
+| Option     | Description                             |
+|:-----------|:----------------------------------------|
+| --warning  | Warning threshold (1min,5min,15min).    |
+| --critical | Critical threshold (1min,5min,15min).   |
+| --average  | Load average for the number of CPUs.    |
+
+</TabItem>
+<TabItem value="Memory" label="Memory">
+
+| Option                   | Description                                                                                                                                                                                                                                                                                          |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --units                  | Units of thresholds (default: '%') ('%', 'absolute')(deprecated. Please use new counters directly)                                                                                                                                                                                                   |
+| --free                   | Apply thresholds on free space instead of on used space. (deprecated. Please use newcounters directly)                                                                                                                                                                                                                      |
```

Comment:
```suggestion
| --free                   | Apply thresholds on free space instead of on used space (deprecated, please use new counters directly).                                                                                                                                                                                                                      |
```